pages/history.js

```diff
@@ -1,10 +1,93 @@
+import {
+	faBirthdayCake,
+	faCircleNotch,
+	faMask,
+	faRandom,
+	faVideo,
+} from "@fortawesome/free-solid-svg-icons";
 import Head from "next/head";
 import styled from "styled-components";
+import Event from "../components/event";
+
+const EventsContainer = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+
+	& > * {
+		flex: 0 1 calc(50% - 16px);
+		margin: 8px;
+
+		max-width: 35%;
+	}
+
+	@media (max-width: 800px) {
+		flex-direction: column;
+
+		& > * {
+			flex-basis: 100%;
+			max-width: unset;
+		}
+	}
+`;
 
 export default () => (
 	<>
 		<Head>
 			<title>History - The Snakeroom</title>
 		</Head>
+		<EventsContainer>
+			<Event icon={faMask} name="Imposter" time="April 2020">
+				In this event, players received four messages by other players
+				mixed with one generated by an AI and needed to determine which
+				one was generated by the bot. The third iteration of the Sneknet
+				collected results from each round to determine which of the five
+				messages was generated by the bot. With enough data, the correct
+				answer could automatically be selected.
+			</Event>
+			<Event icon={faVideo} name="RPAN" time="August 2019">
+				RPAN is a streaming event with a cap on streamers and length.
+				Before its initial launch week, the Snakeroom used a script that
+				sniped slots and allowed users to broadcast from an OBS stream,
+				bypassing the intended limitation of streaming from a mobile
+				device and allowing Snakeroom members to create{" "}
+				<strong>Place Live</strong> and{" "}
+				<strong>RPAN Plays Pokémon</strong>. They were also able to
+				stream prior to the event going live, although Reddit admins
+				quickly shut this down.
```

Comment:
```suggestion
RPAN is a streaming event with a cap on streamers and length.
				The Snakeroom was able to stream before its initial launch 
				although Reddit admins quickly shut this down. 
				Once the event went live, the Snakeroom used a script that 
				sniped slots and allowed users to broadcast from an OBS stream,
				bypassing the intended limitation of streaming from a mobile
				device and allowing Snakeroom members to create{" "}
				<strong>Place Live</strong> and{" "}
				<strong>RPAN Plays Pokémon</strong>.
```

Reply:
This suggestion seems to add a sentence that was already in the event description.